assets/src/domain/eventEditor/ui/datetimes/datesList/actionsMenu/AssignTicketsButton.tsx

```diff
@@ -24,14 +25,17 @@ const AssignTicketsButton: React.FC<EntityListItemProps> = React.memo(({ entity
 		? `${__('Related Tickets:')} ${relatedTicketDbIds.join(', ')}`
 		: __('There are no tickets assigned to this datetime. Please click the ticket icon to update the assignments.');
 
+	const tooltipProps = { placement: 'right' as 'right' };
```

Comment:
![image](https://user-images.githubusercontent.com/1477453/79237038-5ffedb80-7e76-11ea-90b8-99e346805e4f.png)

Reply:
That's TS string to literal casting 😄